locales-pending/premium.ftl

```diff
@@ -166,7 +166,7 @@ dashboard-top-banner-protect-your-data-title = Let’s protect your data
 dashboard-top-banner-protect-your-data-description = We found your data in { $data_breach_total_num } data breaches and { $data_broker_total_num } sites selling your personal info. We’ll guide you on how to fix it.
 dashboard-top-banner-protect-your-data-cta = Let’s fix it
 
-dashboard-top-banner-monitor-protects-your-even-more-title = Monitor now protects you even more
+dashboard-top-banner-monitor-protects-your-even-more-title = { -product-short-name } now protects you even more
```

Comment:
https://github.com/mozilla/blurts-server/blob/4aa64be7cb1457b6da02499f2f3d97601a64660f/locales/en/app.ftl#L8

Was:
> File path: premium.ftl
> Message ID: dashboard-top-banner-monitor-protects-your-even-more-title
> Position: line 169 column 62
> Error (CO01): Strings should use the corresponding terms instead of hard-coded brand names (Monitor)